<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmVolume</class>
 <widget class="QDialog" name="frmVolume">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>780</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>780</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>620</width>
    <height>780</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Create a new encfs encrypted volume</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>601</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Volume settings:</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Unique volume name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_volumename">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_encfsfolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Location of new (empty) encfs folder:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_encfsfolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_selectencfsfolder">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination (mount) folder:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_mountfolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_selectmountfolder">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>601</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Password options:</string>
   </property>
   <widget class="QLineEdit" name="txt_password">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeat password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_password2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_saveinkeychain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save password in keychain</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>601</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>mount options:</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>encfs/gocryptfs mount options:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_encfsmountoptions">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_automount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Automatically mount this volume when application starts</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_preventautounmount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Prevent auto-unmounting this volume on application exit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_accesstoother">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Allow access to 'other' user (you must be root/admin) - needed for Spotlight to work</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_mountaslocal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mount as a local volume</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_usesudo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Use 'sudo' to mount this volume. (sudo must be configured not to require a password!)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>750</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>750</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tab_volumetypes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>601</width>
     <height>221</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <widget class="QWidget" name="tab_gocryptfs_options">
    <attribute name="title">
     <string>gocryptfs options</string>
    </attribute>
    <widget class="QGroupBox" name="grp_gocryptfsoptions">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QCheckBox" name="chk_gocryptfs_plaintext">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Do not encrypt filenames</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_gocryptfs_aessiv">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Use AES-SIV encryption</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_encfs_options">
    <attribute name="title">
     <string>encfs options</string>
    </attribute>
    <widget class="QGroupBox" name="grp_encfsoptions">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Filename encoding:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmb_blocksize">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>70</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Blocksize:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>571</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>encfs profile</string>
      </property>
      <widget class="QRadioButton" name="radio_balanced">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Balanced</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_performance">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Performance</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_secure">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Secure</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_custom">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>30</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Custom</string>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="chk_externaliv">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>170</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>External IV</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_chainediv">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>170</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Chained IV</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmb_filenameencoding">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>181</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Keysize:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_perblockhmac">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Per-block HMAC</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmb_ciphers">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>181</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cipher algorithm:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_perfileuniqueiv">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Per-file unique IV</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmb_keysize">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_volumetypes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>601</width>
     <height>45</height>
    </rect>
   </property>
   <property name="title">
    <string>Volume type:</string>
   </property>
   <widget class="QRadioButton" name="radio_type_encfs">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>encfs</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_type_gocryptfs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>gocryptfs</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
